src/conf.ts

```diff
@@ -27,6 +27,9 @@ export const saveWorkspace = (workspace = 'master') =>
 export const saveEnvironment = (env: Environment) =>
   conf.set('env', env)
 
+export const saveStickyHost = (appId: string, stickyHost: string) =>
+  conf.set(`${appId}.sticky-host`, {stickyHost, lastUpdated: Date.now()})
```

Comment:
I was checking the `vtex.json` file where `configstore` saves the configs. I think configstore is messing up with the `.`'s in your strings and interpreting them as "attributes of an object". The relevant part in my `vtex.json` looks like:
```
		"render-ssr@2": {
			"1": {
				"1": {
					"sticky-host": {
						"stickyHost": "ExFdGkgBTFlfBVdLGVFMUkwHHwoNG1MZCFheSEdCWRQGBFRPAFFbAgBYUAoMXV8FUVEBHlcBB1ZcAV5TAgFB",
						"lastUpdated": 1556212228504
					}
				}
			}
		}
	}

```
Do you think we can make this cleaner? Also, can you remove the app's version before saving the stickyHost, so we can keep the stickyHost across different versions of the same app? I know there is a function somewhere in toolbelt's code that does that.